lib/blocks.php

```diff
@@ -81,6 +81,12 @@ function unregister_block_type( $name ) {
 function do_blocks( $content ) {
 	global $wp_registered_blocks;
 
+	if ( ! function_exists( 'mb_ereg' ) ) {
```

Comment:
These functions are all bundled together in the `mbstring` extension, and `mb_ereg` is the main one that we actually use.  Switched to `extension_loaded( 'mbstring' )` in 92aa724.